www/content/tutorials/cli.md

```diff
@@ -7,3 +7,98 @@ weight = 4
 +++
 
 {{< wip >}}
+
+# tricorder
+This ist the main Command. 
```

Comment:
`ist` vs `is` 🙂

But maybe this sentence is superfluous, the next sentence starts with "This command [...]".

Reply:
I Think you can already guess my mother tongue :)

Reply:
Ich spreche deutsch ein bischen, aber nicht fließend :)

Reply:
done